source/integrate-with-integration-environment/authenticate-your-user.html.md.erb

```diff
@@ -211,16 +211,16 @@ Location: https://YOUR_REDIRECT_URI?error=invalid_request
 &state=1234
 ```
 
-| Error                       | More information about your error                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
-|-----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `unauthorized_client`       | Your service is not authorised to request an authorisation code.<br>Make sure you have [registered your service to use GOV.UK One Login][integrate.register-your-service].                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
Could the error be return in this case as well or will it only be if a client is disabled.

Reply:
No, see https://github.com/govuk-one-login/tech-docs/pull/242#issuecomment-2265427824

Reply:
Understood